packages/rolldown/src/cli/commands/bundle.ts

```diff
@@ -14,13 +19,133 @@ export async function bundle(configExport: RolldownConfigExport) {
 }
 
 async function bundleInner(options: RolldownOptions) {
-  const start = performance.now()
+  const dir = options.output?.dir ?? 'dist'
 
-  const build = await rolldown(options)
+  const startTime = performance.now()
 
+  const build = await rolldown(options)
   const _output = await build.write(options?.output)
 
-  consola.log(
-    `Finished in ${colors.bold((performance.now() - start).toFixed(2))} ms`,
+  const entTime = performance.now()
+
+  const outputEntries = await collectOutputEntries(_output.output)
+  const outputLayoutSizes = collectOutputLayoutSizes(outputEntries)
+  printOutputEntries(outputEntries, outputLayoutSizes, dir)
+
+  consola.success(
+    `Finished in ${colors.bold((entTime - startTime).toFixed(2))} ms`,
   )
 }
+
+type ChunkType = 'chunk' | 'asset'
+type OutputEntry = {
+  type: ChunkType
+  fileName: string
+  size: number
+  compressedSize: number | null
+}
+
+async function collectOutputEntries(
+  output: RollupOutput['output'],
+): Promise<OutputEntry[]> {
+  return await Promise.all(
+    output.map(async (chunk) => {
+      return {
+        type: chunk.type,
+        fileName: chunk.fileName,
+        size: chunk.type === 'chunk' ? chunk.code.length : chunk.source.length,
+        compressedSize: await calculateCompressedSize(
+          chunk.type === 'chunk' ? chunk.code : chunk.source,
+        ),
+      }
+    }),
+  )
+}
+
+async function calculateCompressedSize(
+  code: string | Uint8Array,
+): Promise<number | null> {
+  const compressed = await compress(
```

Comment:
This would really slow down the speed of cli for bigger project with doing real compressing action. Nt sure if this is good idea.